Makefile

```diff
@@ -1,8 +1,13 @@
+# This is not really the source of truth for the build system; refer to
+# cloudbuild.yaml for that.
+
+BAZEL_BUILD_OPTS:=--workspace_status_command=${PWD}/workspace_status.sh
```

Comment:
non-blocking suggestion:
I think we usually put this in a bazelrc, which allows invoking bazel w/o any wrappers.
eg:
https://github.com/kubernetes/test-infra/blob/9a497cc7808fbb19cd85eb8caa1453732272745a/.bazelrc#L2

Reply:
Opened a new issue for this.

Reply:
Errr the issue is #4

Reply:
I agree with Ben's suggestion. Why not just make that change in this PR?

Reply:
@ixdy Because he said it was non-blocking. Also, I didn't want to hold back #2 because this PR has stuff that touches a lot of bazel stuff.